modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/IgnitePdsCacheWalDisabledOnRebalancingTest.java

```diff
@@ -92,16 +90,12 @@
 
     /** {@inheritDoc} */
     @Override protected void beforeTestsStarted() throws Exception {
```

Comment:
There are a lot of methods left that only invoke their "super" counterpart. Can you remove them?